scripts/radiusclient.conf

```diff
@@ -0,0 +1,78 @@
+# General settings
+
+# specify which authentication comes first respectively which
+# authentication is used. possible values are: "radius" and "local".
+# if you specify "radius,local" then the RADIUS server is asked
+# first then the local one. if only one keyword is specified only
+# this server is asked.
+auth_order	radius,local
```

Comment:
do you know what this line is for? I know it's the default but it seems like we should never look locally for logins